.github/workflows/release-native.yml

```diff
@@ -61,20 +61,17 @@ jobs:
   dockerize:
     needs: [ scala-native ]
     if: ${{ github.event_name == 'release' == success() }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.platform == 'linux/arm64' && 'ubuntu-24.04-arm' || 'ubuntu-latest' }}
```

Comment:
not ubuntu-latest anymore, it's a fixed version now.

also, any way to re-use the same deploy matrix (with os and name fields) from above?

Reply:
switched to 22.04. I'll think about reusing matrix, not implemented yet